examples/nginx-https/README.md

```diff
@@ -0,0 +1,99 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<h1>*** PLEASE NOTE: This document applies to the HEAD of the source
+tree only. If you are using a released version of Kubernetes, you almost
+certainly want the docs that go with that version.</h1>
+
+<strong>Documentation for specific releases can be found at
+[releases.k8s.io](http://releases.k8s.io).</strong>
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+# Nginx https service
+
+This example creates a basic nginx https service useful in verifying proof of concept, keys, secrets, and end-to-end https service creation in kubernetes.
+It uses an [nginx server block](http://wiki.nginx.org/ServerBlockExample) to serve the index page over both http and https.
+
+### Generate certificates
+First generate a self signed rsa key and certificate that the server can use for TLS.
+
+```shell
+$ make keys secret KEY=/tmp/nginx.key CERT=/tmp/nginx.crt SECRET=/tmp/secret.json
+```
+
+### Run the nginx https server in a docker container
+
+This step is optional. The [docker daemon](https://docs.docker.com/userguide/) needs to be running on your host for this to work.
+Start the nginxhttps server and mount the generated keys into the container.
+```
+$ docker run -P -d --name mynginx -v "/tmp/nginx.crt:/etc/nginx/ssl/nginx.crt" -v "/tmp/nginx.key:/etc/nginx/ssl/nginx.key" nginxhttps
```

Comment:
It's a little squicky that we have raw docker commands going on.  Can we do without?  I know certs is weird.